### ISSUE TITLE
[FEATURE]I hope Liquidbounce will add blockanimation

### ISSUE THREAD

**Issue by Litely1222** [CCBlueX/LiquidBounce1.8-Issues#3814]
yes...

**Comment by TheFleksisCekcu** [comment]
no

**Comment by aftery142** [comment]
> 
> 
> no

**Comment by ChocoPieisme** [comment]
YES

**Comment by TheFleksisCekcu** [comment]
no

**Comment by ChocoPieisme** [comment]
YES

**Comment by visionfutur** [comment]
it looks like this ? https://streamable.com/f5p25h(note: it's just a visual)

**Comment by Litely1222** [comment]
> it looks like this ? https://streamable.com/f5p25h (note: it's just a visual)
yes i need it :<

**Comment by ChocoPieisme** [comment]
plz add it

**Comment by SenkJu** [comment]
Maybe we could make the already existing SwingAnimation module more customizable.

**Comment by Litely1222** [comment]
> Maybe we could make the already existing SwingAnimation module more customizable.

thank you very much

**Comment by ChocoPieisme** [comment]
very nice, but you should add a module called "BlockAnimation", it will be better

**Comment by visionfutur** [comment]
i don't think if they add it. 
the source code Animations is available  in ccbluex forum. You can check that

**Comment by Litely1222** [comment]
> i don't think if they add it.
> the source code Animations is available in ccbluex forum. You can check that

where

**Comment by ChocoPieisme** [comment]
yes but marco should have it in b73, kinda if I have my custom b73, when b74 release i need to change the src again, and im very lazy

**Comment by visionfutur** [comment]
The animation source is here https://gist.github.com/visionfutur/8a3c2201655d13fda0a70c9bd74c9b69/revisions
